Library/ViewModels/LoginViewModelTests.swift

```diff
@@ -189,43 +189,60 @@ final class LoginViewModelTests: TestCase {
     self.vm.inputs.viewWillAppear()
     self.vm.inputs.onePassword(isAvailable: false)
 
-    self.onePasswordButtonHidden.assertValues([true])
+    self.onePasswordButtonHiddenObserver.assertValues([true])
   }
 
-  func testOnePasswordFlow() {
+  func testOnePasswordButtonHidesBasedOnPasswordAutofillAvailabilityInIOS12AndPlus() {
     self.vm.inputs.viewWillAppear()
     self.vm.inputs.onePassword(isAvailable: true)
 
-    XCTAssertEqual([true, nil],
-                   self.trackingClient.properties(forKey: "1password_extension_available", as: Bool.self))
+    if #available(iOS 12, *) {
+      self.onePasswordButtonHiddenObserver.assertValues([true])
+    } else {
+      self.onePasswordButtonHiddenObserver.assertValues([false])
+    }
+  }
 
-    XCTAssertEqual([nil, true],
-                   self.trackingClient.properties(forKey: "one_password_extension_available", as: Bool.self))
+  func testOnePasswordFlow() {
+    guard #available(iOS 12, *) else {
```

Comment:
Since we don't show the `1Password` button on iOS 12..there's no way to test this flow on that OS version.